packages/discord.js/typings/index.d.ts

```diff
@@ -5310,7 +5310,7 @@ export interface AutoModerationRuleCreateOptions {
   eventType: AutoModerationRuleEventType;
   triggerType: AutoModerationRuleTriggerType;
   triggerMetadata?: AutoModerationTriggerMetadataOptions;
-  actions: AutoModerationActionOptions;
+  actions: AutoModerationActionOptions[];
```

Comment:
You need to update this too:

https://github.com/discordjs/discord.js/blob/429dbccc85cabd9986b2e8bf443bf384e4ddc61a/packages/discord.js/src/structures/AutoModerationRule.js#L231-L239